### PR TITLE
[Partner] Add artworksConnection field

### PR DESCRIPTION
**Pull request by damassi**
Adds a new `artworksConnection` field to `Partner`.

(Need to follow up with Gravity PR for `exclude`) 

```gql
{
  partner(id:"bau-xi-gallery") {
    artworksConnection(first:3) {
      pageInfo {
        hasNextPage
      }
      edges {
        node {
          id
        }
      }
    }
  }
}
```

```json
{
  "data": {
    "partner": {
      "artworksConnection": {
        "pageInfo": {
          "hasNextPage": true
        },
        "edges": [
          {
            "node": {
              "id": "cara-barer-iceberg"
            }
          },
          {
            "node": {
              "id": "david-leventi-rezzonico"
            }
          },
          {
            "node": {
              "id": "virginia-mak-hidden-nature-08"
            }
          }
        ]
      }
    },
  }
}
```

### CODE REVIEW

File: src/lib/loaders/loaders_with_authentication/gravity.ts [hunk 1]
```diff
@@ -6,186 +6,74 @@ import trackedEntityLoaderFactory from "lib/loaders/loaders_with_authentication/
...
-    ),
-    collectionLoader: gravityLoader(id => `collection/${id}`, {
-      user_id: userID,
-    }),
-    collectionArtworksLoader: gravityLoader(
-      id => `collection/${id}/artworks`,
-      { user_id: userID },
-      { headers: true }
-    ),
+    authenticatedArtworkVersionLoader: gravityLoader(id => `artwork_version/${id}`),
```
Comment (damassi) [inline src/lib/loaders/loaders_with_authentication/gravity.ts]: Alphabetize

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+src/lib/loaders/loaders_without_authentication/gravity.ts
+src/lib/loaders/loaders_with_authentication/gravity.ts
```
Comment (damassi) [inline .prettierignore]: Exclude these loader files for readability

**Comment by mzikherman** [comment]
Nice, these are getting boilerplate now!

**Comment by mzikherman** [comment]
LGTM, feel free to mergey whenever.

**Pull request merged**